src/applications/disability-benefits/all-claims/tests/pages/toxicExposure/toxicExposureConditions.unit.spec.jsx

```diff
@@ -13,13 +13,91 @@ import {
   noneAndConditionError,
 } from '../../../content/toxicExposure';
 import formConfig from '../../../config/form';
+import ToxicExposureConditions from '../../../components/ConfirmationFields/ToxicExposureConditions';
 
 describe('Toxic Exposure Conditions', () => {
   const {
     schema,
     uiSchema,
   } = formConfig.chapters.disabilities.pages.toxicExposureConditions;
 
+  it('renders "None claimed" when no toxic exposure conditions are selected', () => {
+    const formData = {
+      toxicExposure: {
+        conditions: {
+          none: true,
+        },
+      },
+      newDisabilities: [{ condition: 'Asthma' }, { condition: 'COPD' }],
+    };
+    const { getByRole, queryByText } = render(
+      <ToxicExposureConditions formData={formData} />,
+    );
+    expect(getByRole('heading', { name: /toxic exposure/i })).to.exist;
+    expect(queryByText(/none claimed/i)).to.exist;
+    expect(queryByText(/asthma/i)).to.be.null;
+    expect(queryByText(/copd/i)).to.be.null;
+  });
+
+  it('renders claimed toxic exposure conditions', () => {
```

Comment:
I'm curious how this and the 'renders only matching claimed conditions' test are different.